### PR TITLE
5

### PR DESCRIPTION
**Pull request by MCodes9**
Ticket #5
PATCH update vote count

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -93,4 +95,20 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(article.votes).toBe(90);
       });
```
Comment (popatre) [inline __tests__/app.test.js:96]: Its just missing a few tests

- [x]  400 inc_vote not an integer e.g. {inc_votes: 'not a number'}

- [x] 200/400 inc_vote key missing - either respond with an error or make no change.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -93,4 +95,20 @@ describe("PATCH /api/articles/:article_id", () => {
...
   });
+  test("400: Responds a bad request error message when passed an invalid endpoint id", () => {
+    return request(app)
+      .get("/api/articles/invalid_id")
```
Comment (popatre) [inline __tests__/app.test.js]: Are these for the patch request, because they still say .get?

**Review by popatre — CHANGES_REQUESTED** [review]

File: __tests__/app.test.js (hunk 1 above)
Comment (MCodes9) [reply __tests__/app.test.js:96]: Not entirely sure what you mean by 200/400 inc_vote key missing - I've interpreted the best I could. I'm sure you'll let me know if I interpreted it incorrectly!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -96,16 +96,39 @@ describe("PATCH /api/articles/:article_id", () => {
...
+      .send(requestBody)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("400: Responds a bad request error message when request body is empty", () => {
+    return request(app)
+      .patch("/api/articles/invalid_id")
+      .send()
```
Comment (Rosem10) [inline __tests__/app.test.js]: So - good interpreting, and you're more or less right, except this should take an empty object, please! I think the comment above was trying to say - the object will still be there, but the key of inc_votes will be missing. Which would leave us with an empty object 👍

**Review by Rosem10 — APPROVED** [review]

File: __tests__/app.test.js (hunk 3 above)
Comment (MCodes9) [reply __tests__/app.test.js]: Ah - got it - thanks! Will make the changes now

**Review by popatre — APPROVED** [review]

**Pull request merged**